api/src/libs/healthCheck.ts

```diff
@@ -36,9 +34,14 @@ const healthCheck = async (type: string, logger: LogWriterType): Promise<Status>
 };
 
 export const runHealthChecks = async (logger: LogWriterType): Promise<StatusResult> => {
+  const url = process.env.API_BASE_URL;
+  if (!url) {
+    logger.LogError("Missing required environment variable: API_BASE_URL");
+    throw new Error("Missing required environment variable: API_BASE_URL");
+  }
```

Comment:
[dust] Not a huge deal, but we don't tend to mention the environment variable names explicitly. If we're going to log here, I'd rather log the problem rather than specifically the environment variable name.
```suggestion
  if (!url) {
    logger.LogError("API URL is undefined");
    throw new Error(API URL is undefined");
  }
```